### PR TITLE
feat(db): add more templates to the dashboard

### PR DESCRIPTION
**Pull request by ghuntley**
Related to https://github.com/gitpod-io/website/pull/825/ but can be merged seperately.

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **alextugarev** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @alextugarev` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/gitpod-db/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-db/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/gitpod-db/src/typeorm/migration/1628245953986-UpdateExamples.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+             { url: 'https://github.com/gitpod-io/template-golang-cli', description: 'A CLI template, written in Go.', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-rust-cli', description: 'A CLI template, written in Rust.', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-dotnet-core-cli-csharp', description: 'A CLI starter for .NET written in C#.', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-sveltejs', description: 'A Svelte.js app writtten in JavaScript.', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-sveltejskit', description: 'A SvelteKit app template.', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-datasette', description: 'A Datasette template.', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-nix', description: 'A nix template for reproducible development environments.', priority: priority-- },
```
Comment (csweichel) [inline components/gitpod-db/src/typeorm/migration/1628245953986-UpdateExamples.ts:28]: isn't reproducible development environments what Gitpod is about?

Comment (ghuntley) [reply components/gitpod-db/src/typeorm/migration/1628245953986-UpdateExamples.ts:28]: Nix is a _really_ is reproducible - https://r13y.com/ whereas Docker pretends to be (`RUN sudo apt-get upgrade` == fail). What verbiage should be use instead?

Comment (ghuntley) [reply components/gitpod-db/src/typeorm/migration/1628245953986-UpdateExamples.ts:28]: proposal: `A template for reproducible development environments via nix.`

**Comment by ghuntley** [comment]
Looks good.

/assign @alextugarev

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **alextugarev** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/gitpod-db/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-db/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request closed without merging**